src/test/java/reposense/authorship/AnnotatorAnalyzerTest.java

```diff
@@ -0,0 +1,69 @@
+package reposense.authorship;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import reposense.authorship.model.FileInfo;
+import reposense.authorship.model.FileResult;
+import reposense.authorship.model.LineInfo;
+import reposense.model.Author;
+import reposense.parser.InvalidLocationException;
+import reposense.template.GitTestTemplate;
+import reposense.util.TestUtil;
+
+public class AnnotatorAnalyzerTest extends GitTestTemplate {
+
+    private static final Author MAIN_AUTHOR = new Author(MAIN_AUTHOR_NAME);
+    private static final Author FAKE_AUTHOR = new Author(FAKE_AUTHOR_NAME);
+    private static final Date SINCE_DATE = TestUtil.getSinceDate(2018, Calendar.FEBRUARY, 8);
+    private static final Date UNTIL_DATE = TestUtil.getUntilDate(2018, Calendar.FEBRUARY, 10);
+
+    @Before
+    public void before() throws InvalidLocationException {
+        super.before();
+        config.setSinceDate(SINCE_DATE);
+        config.setUntilDate(UNTIL_DATE);
+    }
+
+    @Test
+    public void applyAnnotationTest() {
```

Comment:
This was from what we removed previously? Perhaps can apply our test method naming conventions this time

Reply:
Done